tests/load/test_update_utils.py

```diff
@@ -1,5 +1,274 @@
+import unittest
+import pytest
+
 from scribe_data import utils
 
 
-def test_get_language_qid():
-    assert utils.get_language_qid("french") == "Q150"
+def test_get_scribe_languages():
+    test_case = unittest.TestCase()
+
+    # test for content, not order
+    test_case.assertCountEqual(
```

Comment:
You'd asked about this in the commit, and to me it's 100% fine! Thank you for giving this some though!